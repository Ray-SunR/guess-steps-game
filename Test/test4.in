game 140 14 test4A1.txt test4B1.txt
game 55 7 test4A2.txt test4B2.txt
quit
