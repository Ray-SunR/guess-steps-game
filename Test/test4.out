Total is 140, max deduction is 14
Player A's move
Total is 139
Player B's move
Total is 137
Player A's move
Total is 134
Player B's move
Total is 130
Player A's move
Total is 125
Player B's move
Total is 119
Player A's move
Total is 112
Player B's move
Total is 104
Player A's move
Total is 95
Player B's move
Total is 85
Player A's move
Total is 74
Player B's move
Total is 62
Player A's move
Total is 49
Player B's move
Total is 35
Player A's move
Total is 21
Player B's move
Total is 7
Player A's move
Total is 6
Player B's move
Total is 4
Player A's move
Total is 1
Player B's move
Total is -3
Player B wins
Score is
A 0
B 1
Total is 55, max deduction is 7
Player B's move
Total is 48
Player A's move
Total is 41
Player B's move
Total is 34
Player A's move
Total is 27
Player B's move
Total is 21
Player A's move
Total is 16
Player B's move
Total is 12
Player A's move
Total is 9
Player B's move
Total is 7
Player A's move
Total is 6
Player B's move
Total is 4
Player A's move
Total is 1
Player B's move
Total is -3
Player B wins
Score is
A 0
B 2
